[2021-03-07 19:02:45,926] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: save_to_clickhouse_3.log 2021-03-06T00:00:00+00:00 [queued]>
[2021-03-07 19:02:45,933] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: save_to_clickhouse_3.log 2021-03-06T00:00:00+00:00 [queued]>
[2021-03-07 19:02:45,933] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:02:45,933] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-07 19:02:45,933] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-07 19:02:45,949] {taskinstance.py:1063} INFO - Executing <Task(_PythonDecoratedOperator): log> on 2021-03-06T00:00:00+00:00
[2021-03-07 19:02:45,985] {standard_task_runner.py:52} INFO - Started process 48 to run task
[2021-03-07 19:02:45,988] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'save_to_clickhouse_3', 'log', '2021-03-06T00:00:00+00:00', '--job-id', '134', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/save_to_clickhouse_3/main.py', '--cfg-path', '/tmp/tmpwjyex487', '--error-file', '/tmp/tmp0ai92p5m']
[2021-03-07 19:02:45,988] {standard_task_runner.py:77} INFO - Job 134: Subtask log
[2021-03-07 19:02:46,051] {logging_mixin.py:104} INFO - Running <TaskInstance: save_to_clickhouse_3.log 2021-03-06T00:00:00+00:00 [running]> on host 020f66ce59e6
[2021-03-07 19:02:46,097] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=save_to_clickhouse_3
AIRFLOW_CTX_TASK_ID=log
AIRFLOW_CTX_EXECUTION_DATE=2021-03-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-06T00:00:00+00:00
[2021-03-07 19:02:46,097] {logging_mixin.py:104} INFO - Should be saved in clickhouse
[2021-03-07 19:02:46,097] {logging_mixin.py:104} INFO - None
[2021-03-07 19:02:46,106] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=save_to_clickhouse_3, task_id=log, execution_date=20210306T000000, start_date=20210307T190245, end_date=20210307T190246
[2021-03-07 19:02:46,128] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-07 19:02:46,165] {local_task_job.py:146} INFO - Task exited with return code 0
